portal/integrated-windows-authentication/README.md

```diff
@@ -1,47 +1,41 @@
-# Integrated windows authentication
+# Integrated Windows Authentication
```

Comment:
I'm unsure whether this has to be sentence case or title case. IWA is a term, so I think title case is correct though

Reply:
yes I had the same initial uncertainty! But indeed: it's correct use is title case.